ops/ccs-ops-misc/load_test/README.md

```diff
@@ -131,35 +69,33 @@ Essentially, all the items you would set up in the config file are set in a sing
 | `comp_tag` | No | Must be non-empty, consisting of letters, numbers and the `_` character | Tag from which comparison statistics will be loaded from the given `store`. Defaults to `store_tag` if unspecified. |
 | `athena_tbl` | Yes, _if `store` is `s3` **and** `compare` is set_ | N/A | Name of the table to query using Athena if `store` is `s3` and `compare` is set. |
 
-### Quick Run
+### Running via `locust` binary
 
-Once you've run a test once and the configuration file is set, you can "quick run" a test by calling locust directly, like this:
+If the test suite to run will _not_ be bottlenecked by the local machine's single-threaded performance (which is _very_ likely the case, as its atypical for the tests themselves to be the performance bottleneck), then running tests via Locust's binary is the best choice:
 
 ```
-locust -f <path/to/test/file.py>
+locust -f <locustfile> <command-line-args>...
 ```
 
-This will run the test with the parameters of the last test that was run. If you want to set up a configuration file manually, a sample file is included in this directory to copy and modify; simply copy the file and remove -sample from the name so its named config.yml.
+It is recommended to always include the `--headless` command-line argument so that Locust [runs in "headless" mode](https://docs.locust.io/en/stable/quickstart.html#direct-command-line-usage-headless), without its included Web UI:
```

Comment:
I think that's one of the things moving to using Locust built-ins gets us -- we should be able to run tests headful without problems now (excluding data management, which we'll need to revise sometime down the line anyways). I don't think running it headful is particularly useful for the suite of tests we have now, but I feel like it might have its uses for the load tests since that suite will be manually kicked-off and monitored (which the Locust web UI is great for).

Reply:
Running in headful mode spins up a web server on the box to control Locust. I don't think we necessarily need to change from headless servers to support that, just open up whatever port the Locust web server listens on so that we can reach it from the browser -- if we decided that it'd be useful to actually _run_ headful, that is.